src/Components/Blazor/Validation/src/BlazorCompareAttribute.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System.ComponentModel.DataAnnotations
+{
+    /// <summary>
+    /// A <see cref="ValidationAttribute"/> that compares two properties
+    /// </summary>
+    public sealed class BlazorCompareAttribute : CompareAttribute
```

Comment:
Is this because the original CompareAttribute doesn't include the member name? 

If this is something that users are expected to use in their models, can we find a better name for this? BlazorCompareAttribute is highly confusing.

Reply:
> Is this because the original CompareAttribute doesn't include the member name?

Yup. I'm open to suggestions. I kinda went with a whimsical name to indicate this is meant to go away at some point.